docs/reference/cli-reference.md

```diff
@@ -57,11 +57,12 @@ $ infra login --key 1M4CWy9wF5.fAKeKEy5sMLH9ZZzAur0ZIjy
 ### Options
 
 ```
-      --key string        Login with an access key
-      --no-agent          Skip starting the Infra agent in the background
-      --non-interactive   Disable all prompts for input
-      --provider string   Login with an identity provider
-      --skip-tls-verify   Skip verifying server TLS certificates
+      --key string                   Login with an access key
+      --no-agent                     Skip starting the Infra agent in the background
+      --non-interactive              Disable all prompts for input
+      --provider string              Login with an identity provider
+      --skip-tls-verify              Skip verifying server TLS certificates
+      --tls-trusted-cert file path   TLS certificate or CA used by the server
```

Comment:
I'd like to be able to pass a [certificate fingerprint](https://ldapwiki.com/wiki/Certificate%20Fingerprint) so I don't have to always have a file available.

Reply:
That's definitely another option. I can add another flag for `--tls-trusted-fingerprint`.

Reply:
Added that flag in the latest commit